spec/services/order_syncer_spec.rb

```diff
@@ -1,3 +1,5 @@
+require 'spec_helper'
+
 describe OrderSyncer do
```

Comment:
Isn't `OrderSyncer` a bit too core to make it pending? Doesn't it play an important role in placing orders or it isn't related to order placement?

Reply:
I believed the docs, it's subscriptions specific:
# Responsible for ensuring that any updates to a Subscription are propagated to any
# orders belonging to that Subscription which have been instantiated